src/CommunityToolkit.Maui.MediaElement/Handlers/MediaElementHandler.shared.cs

```diff
@@ -19,7 +19,7 @@ public partial class MediaElementHandler
 	/// <summary>
 	/// The default property mapper for this handler.
 	/// </summary>
-	public static IPropertyMapper<MediaElement, MediaElementHandler> PropertyMapper = new PropertyMapper<MediaElement, MediaElementHandler>(ViewMapper)
+	public static readonly IPropertyMapper<MediaElement, MediaElementHandler> PropertyMapper = new PropertyMapper<MediaElement, MediaElementHandler>(ViewMapper)
```

Comment:
Breaking Change

This is unlikely to impact many devs, but it does break the following code:
```cs
MediaElementHandler.PropertyMapper.Add(...)
```